docs/api/crossplaneio/stack-aws/database-aws-crossplane-io-v1beta1.md

```diff
@@ -385,7 +385,6 @@ Name | Type | Description
 `iops` | Optional int | IOPS is the amount of Provisioned IOPS (input/output operations per second) to be initially allocated for the DB instance. For information about valid IOPS values, see see Amazon RDS Provisioned IOPS Storage to Improve Performance (http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS) in the Amazon RDS User Guide. Constraints: Must be a multiple between 1 and 50 of the storage amount for the DB instance. Must also be an integer multiple of 1000. For example, if the size of your DB instance is 500 GiB, then your IOPS value can be 2000, 3000, 4000, or 5000.
 `kmsKeyId` | Optional string | KMSKeyID for an encrypted DB instance. The KMS key identifier is the Amazon Resource Name (ARN) for the KMS encryption key. If you are creating a DB instance with the same AWS account that owns the KMS encryption key used to encrypt the new DB instance, then you can use the KMS key alias instead of the ARN for the KM encryption key. Amazon Aurora Not applicable. The KMS key identifier is managed by the DB cluster. For more information, see CreateDBCluster. If the StorageEncrypted parameter is true, and you do not specify a value for the KMSKeyID parameter, then Amazon RDS will use your default encryption key. AWS KMS creates the default encryption key for your AWS account. Your AWS account has a different default encryption key for each AWS Region.
 `licenseModel` | Optional string | LicenseModel information for this DB instance. Valid values: license-included | bring-your-own-license | general-public-license
-`masterUserPassword` | Optional string | MasterUserPassword is the password for the master user. The password can include any printable ASCII character except &#34;/&#34;, &#34;&#34;&#34;, or &#34;@&#34;. Amazon Aurora Not applicable. The password for the master user is managed by the DB cluster. For more information, see CreateDBCluster. MariaDB Constraints: Must contain from 8 to 41 characters. Microsoft SQL Server Constraints: Must contain from 8 to 128 characters. MySQL Constraints: Must contain from 8 to 41 characters. Oracle Constraints: Must contain from 8 to 30 characters. PostgreSQL Constraints: Must contain from 8 to 128 characters.
```

Comment:
do we do anything for deprecated / removed fields, like mention them in the release notes?

Reply:
This might actually be being removed by mistake - I don't recall us deciding to remove this field. @muvaf does this ring a bell?

Reply:
This field was apparently present but unused, it was removed in this PR https://github.com/crossplaneio/stack-aws/pull/65

Reply:
It seems I didn't update the docs after that change https://github.com/crossplaneio/stack-aws/pull/65

How can we make sure that the doc generation is a part of `make reviewable`?

Reply:
Was taking a look at this over the weekend and I have opened an issue to track the work https://github.com/crossplaneio/crossplane/issues/1166

Reply:
@muvaf if `masterUserPassword` isn't used then what credentials are supported for use in conjunction with `masterUsername`?